### PR TITLE
Add "Get Help" page

### PR DESCRIPTION
**Pull request by Tabrarian**
Adding content outline and basic info to create a "Get Help" page on the 2.0 site.

### CODE REVIEW

**Comment by mirons-google** [comment]
@Tabrarian do you mind combining your planned changes for the /community page to this PR as well? That we we can look at them together and de-dup and ensure the content is consistent.

**Comment by Tabrarian** [comment]
Following on yesterday's discussion, I created a couple includes, updated the community page, and added a help page for use and develop. I'm unsure if I got the include syntax correct. PTAL, thanks!

File: _includes/docs/latest/get_help_bug.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  * What happened? What did you expect to happen?
+  * How can we reproduce the problem?
+  * Which interface are you using (Cloud Platform Console, gcloud)?
+  * Which module are you using?
+
+If you have an idea for a Forseti feature that isn't already on the
+[forseti-security](https://github.com/GoogleCloudPlatform/forseti-security/issues)
+issue list, we'd like to hear from you! Please file a new feature request to the
+issue list so community members and Google engineers can review it.
```
Comment (mirons-google) [inline _includes/docs/latest/get_help_bug.md]: "... so community members and forseti security engineers can review it."

File: _docs/_latest/use/get-help.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Get Help
+order: 010
+---
+
+# {{ page.title }}
+
+This page includes resources if you need help while following the
+[guides]({% link _docs/latest/use/index.md %} to use Forseti.
```
Comment (mirons-google) [inline _docs/_latest/use/get-help.md]: Missing your closing parens after the %}

**Comment by Tabrarian** [comment]
I think I got it, thanks, just waiting on Travis!

**Pull request merged**